airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -53,8 +53,6 @@
   public static final String CONFIG_DATABASE_PASSWORD = "CONFIG_DATABASE_PASSWORD";
   public static final String CONFIG_DATABASE_URL = "CONFIG_DATABASE_URL";
   public static final String RUN_DATABASE_MIGRATION_ON_STARTUP = "RUN_DATABASE_MIGRATION_ON_STARTUP";
-  public static final String LOG_LEVEL = "LOG_LEVEL";
```

Comment:
this is only used in the log4j configuration - leaving this here seemed more confused than not.